namer/marathon/src/main/scala/io/l5d/marathon.scala

```diff
@@ -40,6 +42,7 @@ case class marathon(
     case None => 80
   }
   private[this] def getUriPrefix = uriPrefix.getOrElse("")
+  private[this] def getTtl = ttlMs.getOrElse(250).millis
```

Comment:
250 is probably not actually a good default.  Can we default at like.. 5000?

Also, we probably want to add some sort of jitter so that all instances don't hammer simultaneously... (can be done in a followup branch)